Description: Spelling fixes
Author: Stephen Kitt <skitt@debian.org>

--- a/README.md
+++ b/README.md
@@ -71,7 +71,7 @@
 `g810-led -pp < profilefile # Load a profile`</br>
 `echo -e "k w ff0000\nk a ff0000\nk s ff0000\nk d ff0000\nc" | g810-led -pp # Set multiple keys`</br>
 
-## Testing unsuported keyboards :</br>
+## Testing unsupported keyboards :</br>
 Start by retrieving the VendorID and the ProductID of your keyboard using lsusb.</br>
 `lsusb`</br>
 Sample return :<br>
--- a/src/helpers/help.cpp
+++ b/src/helpers/help.cpp
@@ -79,7 +79,7 @@
 		cout<<"  -dv\t\t\t\t\tDevice vendor ID, such as 046d for Logitech. Can be omitted to match any vendor ID"<<endl;
 		cout<<"  -dp\t\t\t\t\tDevice product ID, such as c337 for Logitech G810. Can be omitted to match any product ID"<<endl;
 		cout<<"  -ds\t\t\t\t\tDevice serial number, Can be omitted to match the first device found"<<endl;
-		cout<<"  -tuk\t\t\t\t\tTest unsuported keyboard with one of supported protocol (1-3) -dv and -dp are required"<<endl;
+		cout<<"  -tuk\t\t\t\t\tTest unsupported keyboard with one of supported protocol (1-3) -dv and -dp are required"<<endl;
 		cout<<endl;
 		cout<<"Values:"<<endl;
 		if((features | KeyboardFeatures::rgb) == features)
@@ -318,7 +318,7 @@
 			cout<<"echo -e \"k w ff0000\\nk a ff0000\\nk s ff0000\\nk d ff0000\\nc\" | g810-led -pp # Set multiple keys"<<endl;
 			cout<<endl;
 		}
-		cout<<"Testing an unsuported keyboard :"<<endl;
+		cout<<"Testing an unsupported keyboard :"<<endl;
 		cout<<"lsusb"<<endl;
 		cout<<"#Sample result of lsusb : ID 046d:c331 Logitech, Inc. (dv=046d and dp=c331)"<<endl;
 		cout<<cmdName<<" -dv 046d -dp c331 -tuk 1 -a 000000"<<endl;
